lib/v2/acs/journal.js

```diff
@@ -0,0 +1,73 @@
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const config = require('@/config').value;
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id ?? '';
+
+    const rootUrl = 'https://pubs.acs.org';
+    const currentUrl = `${rootUrl}/toc/${id}/current?sortBy=DATE`;
```

Comment:
Should use `${rootUrl}/toc/${id}/0/0` as it is the url for ASAP.

Reply:
Thanks 😃